modules/web-console/e2e/testcafe/components/Table.js

```diff
@@ -16,12 +16,13 @@
 
 import {Selector, t} from 'testcafe';
 
-const findCell = Selector((table, rowIndex, columnLabel) => {
-    const _findRowElement = (table, containerSelector) => {
-        const columnIndex = [].constructor.from(
+const findCell = Selector((table, rowIndex, columnId) => {
```

Comment:
Why did you change selector from label to id? I think human readable label is much easier to reason about. Was it causing issues for columns without text in header?

Reply:
We have columns with icon only.